src/main/resources/scripts/-examples/chest menus.sk

```diff
@@ -0,0 +1,55 @@
+
+#
+# An example of opening a chest inventory with an item.
+# Players will be able to take the item out.
+#
+
+command /simple chest:
+    permission: skript.example.chest
+    trigger:
+        set {_chest} to a new chest inventory named "Simple Chest"
+        set slot 0 of {_chest} to apple # Slots are numbered 0, 1, 2...
```

Comment:
We should be opening `{_chest}` to the player after this; the above comment says it's an example of *opening* a chest inventory but it's not doing so.